### ISSUE TITLE
Incompatible geth version

### ISSUE THREAD

**Issue by alyyousuf7** [golemfactory/clay#1403]
I followed the instructions from [this link](https://github.com/golemfactory/golem/wiki/Installation-on-Ubuntu) and ran `golemapp --nogui`. This was the output:

```bash
λ golemapp --nogui

Failed to import scrypt. This is not a fatal error but does
mean that you cannot create or decrypt privkey jsons that use
scrypt

WARNING  [golem.appconfig                    ] Can't open file /home/ali/.local/share/golem/SimpleEnv/minilight.ini: [Errno 2] No such file or directory: '/home/ali/.local/share/golem/SimpleEnv/minilight.ini'
Traceback (most recent call last):
  File "/Projects/golem/golemapp.py", line 124, in <module>
  File "/Projects/golem/venv/lib/python3.5/site-packages/click/core.py", line 722, in __call__
  File "/Projects/golem/venv/lib/python3.5/site-packages/click/core.py", line 697, in main
  File "/Projects/golem/venv/lib/python3.5/site-packages/click/core.py", line 895, in invoke
  File "/Projects/golem/venv/lib/python3.5/site-packages/click/core.py", line 535, in invoke
  File "/Projects/golem/golemapp.py", line 91, in start
  File "/Projects/golem/golem/node.py", line 26, in __init__
  File "/Projects/golem/golem/client.py", line 155, in __init__
  File "/Projects/golem/golem/transactions/ethereum/ethereumtransactionsystem.py", line 36, in __init__
  File "/Projects/golem/golem/ethereum/client.py", line 19, in __init__
  File "/Projects/golem/golem/ethereum/node.py", line 93, in __init__
OSError: Incompatible geth version: 1.7. Expected >= 1.6.1 and <= 1.6.999
[19888] Failed to execute script golemapp
λ 
```

Need to update installation script to install compatible `geth` version.

**Comment by chadcampbell** [comment]
I'm experiencing the same issue. I ran into this issue on a fresh installation of Ubuntu.

**Comment by badb** [comment]
This should be fixed in 0.8.1 & on develop.
